lib/faker/date.ex

```diff
@@ -0,0 +1,43 @@
+if Version.match?(System.version(), ">= 1.3.0") do
+  defmodule Faker.Date do
+    @moduledoc """
+    Functions for generating dates
+    """
+
+    @seconds_per_day 86400
+
+    @doc """
+    Returns a random date of birth for a person aged between ~12 to ~99
+    """
+    @spec date_of_birth() :: Date.t
+    def date_of_birth do
```

Comment:
What do you think if we'll add 3 variations?

- default 18..99
- 16 - exact
- 18..27 - in between